projects/control-service/projects/pipelines_control_service/src/main/resources/application.properties

```diff
@@ -143,3 +143,9 @@ mail.smtp.host=smtp.vmware.com
 # Correspond to those defined by kubernetes
 # See https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy
 datajobs.deployment.builder.imagePullPolicy=IfNotPresent
+
+#Settings for Kerberos authorization provider.
+datajobs.properties.kerberosPrincipal=""
```

Comment:
We need to decide which properties to expose as an env variable/helm chart and property names.

Reply:
We'd need to extend [security section of the helm chart](https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml#L306)

We can and should discuss this in separate PR.

Reply:
My suggestion is to create PR with modified values.yaml with the new options described (and their description) only without any implementation (yet).

Reply:
https://github.com/vmware/versatile-data-kit/pull/764